data/mapcss-mapping.csv

```diff
@@ -829,7 +829,7 @@ barrier|block;714;
 barrier|toll_booth;715;
 historic|memorial;716;
 historic|ruins;717;
-historic|monument;718;
+historic|monument;[historic=monument],[man_made=obelisk];;;;718;
```

Comment:
maybe it'd be better to do this in replaced_tags.txt?

Reply:
Here we can understand what goes where at a glance in one place. With replaced_tags.txt we have to refer to another file always, its inconvenient and is easy to overlook.

I think after https://github.com/organicmaps/organicmaps/issues/7728 we can get rid of replaced_tags.txt altogether and have everything nicely and conveniently in one file.